test/llvmpasses/alloc-opt.jl

```diff
@@ -0,0 +1,204 @@
+# RUN: julia --startup-file=no %s | opt -load libjulia.so -AllocOpt -S - | FileCheck %s
+
+## WIP
+
+isz = sizeof(UInt) == 8 ? "i64" : "i32"
```

Comment:
We should really not have the pass depend on the how architecture (and have it look at the target arch/triple instead).